source/default/_patterns/01-atoms/button/_button-dropdown-split.twig

```diff
@@ -1,18 +1,41 @@
 {#
-  Split Button Dropdown!
+/**
+ * @file Split Button Dropdown!
+ * Documenting Split Button Examples from Bootstrap.
+ *
+ * Available variables:
+ * - button_element: [string]: button|a|input choose the element type for your button.
```

Comment:
Let's make an indication that all the props here come from a parent so we don't have to deal with dupe prop documentation.